tests/phpunit/CRM/Utils/TokenTest.php

```diff
@@ -60,8 +60,8 @@ public function testSearchByPrimaryLocation() {
    * Test for replaceGreetingTokens.
    *
    */
-  public function testReplaceGreetingTokens() {
-    $tokenString = 'First Name: {contact.first_name} Last Name: {contact.last_name} Birth Date: {contact.birth_date} Prefix: {contact.prefix_id} Suffix: {contact.individual_suffix}';
+  public function testReplaceGreetingTokens(): void {
+    $tokenString = 'First Name: {contact.first_name} Last Name: {contact.last_name} Birth Date: {contact.birth_date} Prefix: {contact.prefix_id:label} Suffix: {contact.individual_suffix}';
```

Comment:
fully removes 'prefix_id' since it really is out of date & ambigous - also in the tests